### PR TITLE
Fixed Uncaught ReferenceError caused by Keen being blocked by AdBlock

### PR DESCRIPTION
**Pull request by kotborealis**
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: js/everything.js [hunk 1]
```diff
@@ -428,10 +432,15 @@ function pickDemo() {
...
 $(function() {
     // Keen.io
-    client = new Keen({ // eslint-disable-line no-undef
-        projectId: "5690c384c1e0ab0c8a6c59c4",
-        writeKey: "630fa16847ce5ffb01c9cc00327498e4e7716e0f324fb14fdf0e83ffc06f9eacff5fad1313c2701efe4a91c88c34b8d8153cbb121c454056bb63caf60a46336dd9c9e9855ecc5202ef3151d798eda40896d5111f44005c707cbfb32c7ae31070d129d6f520d5604fdbce5ad31e9c7232"
-    });
+    if (Keen === undefined) { // eslint-disable-line no-undef
+        console.warn("Apparently, Keen.io was blocked by AdBlock.");
+    }
+    else {
```
Comment (shakeelmohamed) [inline js/everything.js]: I'd rather just have `if (Keen)`, the log isn't necessary IMO

File: js/everything.js [hunk 2]
```diff
@@ -12,6 +12,10 @@ function anonymizeFileUrl() {
 }
 
 function sendKeenEvent(_msg, _data) {
+    if (client === undefined) {
```
Comment (shakeelmohamed) [inline js/everything.js]: I'd rather just return, no need for a log message

```
if (!client) {
    return;
}

**Comment by shakeelmohamed** [comment]
Thanks @kotborealis! A couple minor changes and this will be good to go

File: js/everything.js [hunk 3]
```diff
@@ -428,10 +431,12 @@ function pickDemo() {
 
 $(function() {
     // Keen.io
-    client = new Keen({ // eslint-disable-line no-undef
-        projectId: "5690c384c1e0ab0c8a6c59c4",
-        writeKey: "630fa16847ce5ffb01c9cc00327498e4e7716e0f324fb14fdf0e83ffc06f9eacff5fad1313c2701efe4a91c88c34b8d8153cbb121c454056bb63caf60a46336dd9c9e9855ecc5202ef3151d798eda40896d5111f44005c707cbfb32c7ae31070d129d6f520d5604fdbce5ad31e9c7232"
-    });
+    if (typeof Keen !== "undefined") { // eslint-disable-line no-undef
```
Comment (shakeelmohamed) [inline js/everything.js]: Can we change this to `if (Keen) {`?

Comment (kotborealis) [inline js/everything.js]: screenshot

**Comment by shakeelmohamed** [comment]
Thanks @kotborealis, welcome to the org! 🎉

**Pull request merged**